test-resources/openapi-service-specs/swagger-yaml-service.yml

```diff
@@ -54,3 +54,23 @@ definitions:
         type: integer
         description: An integer property
         required: false
+  TestEntity_2:
```

Comment:
[nth] Consider giving those better names (what is the test case those entities are used for?)